reaper.go

```diff
@@ -28,7 +28,7 @@ type reaperContextKey string
 
 var (
 	dockerHostContextKey = reaperContextKey("docker_host")
-	reaper               *Reaper // We would like to create reaper only once
+	reaperSingleton      *Reaper // We would like to create reaper only once
```

Comment:
Wdyt if we call this variable as `reaperInstance`. I know that singleton is a well-known word in the industry, but in the comments we always use the instance word: https://github.com/testcontainers/testcontainers-go/pull/782/files#diff-e8ab4ec17e978db949cd15726ba0efa9c89fd714e7e6e0aa537ed7fc1bbde7fdR48

Reply:
sure, will rename